oneview/resource_fc_network.go

```diff
@@ -89,6 +89,19 @@ func resourceFCNetwork() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"scopesUri": {
```

Comment:
managedSanUri attribute is missing if we are adding everything

Reply:
Added the ManagedSanUri attribute